src/routes/InventoryComponents/InventoryPageHeader.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import {
+  PageHeader,
+  PageHeaderTitle,
+} from '@redhat-cloud-services/frontend-components/PageHeader';
+import { DesktopIcon } from '@patternfly/react-icons';
+import useFeatureFlag from '../../Utilities/useFeatureFlag';
+import { pageContents } from '../InventoryPage';
+import { AccountStatContext } from '../../Routes';
+import FontAwesomeImageIcon from '../../components/FontAwesomeImageIcon';
+
+const InventoryContentToggle = ({ changeMainContent, mainContent }) => (
+  <ToggleGroup aria-label="Inventory content toggle">
+    <ToggleGroupItem
+      icon={<DesktopIcon />}
+      aria-label="Hybrid inventory"
+      isSelected={mainContent === pageContents.hybridInventory.key}
+      onChange={() => changeMainContent(pageContents.hybridInventory.key)}
+    />
+    <ToggleGroupItem
+      icon={<FontAwesomeImageIcon />}
+      aria-label="Bifrost"
+      isSelected={mainContent === pageContents.bifrost.key}
+      onChange={() => changeMainContent(pageContents.bifrost.key)}
+    />
+  </ToggleGroup>
+);
+
+const InventoryPageHeader = (toggleProps) => {
+  const isBifrostEnabled = useFeatureFlag('hbi.api.disable-xjoin');
+  const { hasBootcImages } = useContext(AccountStatContext);
```

Comment:
Is it worth it to provide `isBifrostEnabled` via the AccountStatContext as well?

Reply:
The context is used to hold information about an account. I believe the flag would not be a proper thing to add.